.github/workflows/sandbox-rebuild.yml

```
@@ -0,0 +1,45 @@
+name: SandBox Rebuild
+
+on: push
+
+env:
+  AWS_REGION: ${{ vars.AWS_REGION }}
+  BRANCH_NAME: ${{ github.head_ref || github.ref_name }} 
```

yamllint (1.29.0-1)

[error] trailing spaces

(trailing-spaces)